README.md

```
@@ -171,8 +224,18 @@ El dashboard estará disponible en `http://localhost:3001`.
 
 ## Estructura del Proyecto
 
-- `src/`: Código fuente del cliente Python.
-- `dashboard/`: Código fuente de la aplicación Next.js.
-- `questdb/`: Datos persistentes de la base de datos (generado al ejecutar).
-- `sql/`: Scripts de inicialización de base de datos.
-- `secrets/`: Credenciales y ficheros sensibles.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)